coalib/results/result_actions/OpenEditorAction.py

```diff
@@ -63,6 +63,10 @@
         'args': '--wait',
         'gui': True
     },
+    'brackets': {
+        'file_arg_template': '{filename}',
```

Comment:
Can you please explain you reason for this change. Why is it only `{filename}`?

Reply:
Never mind. Found my answers at https://github.com/adobe/brackets/wiki/Command-Line-Arguments#linux. Thanks for the reference. This PR looks good to me now :)

Reply:
This is good enough. Even passing the filename wouldn't load the file, it would simply just open the editor.